### ISSUE TITLE
Ensure we capture CITE seq and cell hashing metadata

### ISSUE THREAD

**Issue by hewgreen** [HumanCellAtlas/metadata-schema#759]
Given the excitement surrounding this technology and adoption by 10X in v3 we need to ensure we are capturing the extra metadata to describe the barcodes and antibodies used.

We have already ingested a dataset of this type and need to ensure it is well described. We should also enhance the metadata model so we can describe these methods in the future.

We should investigate:
- [ ] How should we distinguish between cell hashing and CITE seq experiments?
- [ ] How should antibody metadata be stored?
- [ ] How should experiments be demultiplexed before ingestion?
- [ ] Which barcodes should we store
- [ ] Is there any other metadata we should collect to describe this data type?

**Comment by malloryfreeberg** [comment]
@willrockout I think there is a lot of pressure to get this done sooner rather than later. There is the one dataset already in production that uses this technique, and the wranglers have been hearing that groups are working on this and related techniques. Do you think you can have this ticket done in the next 2 weeks? If not, can you please find someone else who can work on it? If you need help clarifying the acceptance criteria for considering this ticket "done", please ask.

**Comment by zperova** [comment]
@willrockout we just got another Cite-seq dataset

**Comment by malloryfreeberg** [comment]
Copying some relevant info from a recent email thread:

From @samanehsan : 

> In the latest [pull request](https://github.com/HumanCellAtlas/lira/pull/133), I added a clause to exclude any experiments using CITE-seq as the sequencing approach. As it was mentioned during code review, this is not a perfect solution since it could still match other assays that we cannot process.
> At a recent Broad demo, Tim mentioned there are differences between the assay preparation for 10x v2 sequencing and CITEseq experiments that we could use to differentiate between the two. Does anyone know what values we could use from the assay preparation metadata that are specific to 10x v2 sequencing? 
> The goal here is to make sure the inclusion criteria of the query is specific enough to match only 10x v2 single-cell sequencing data without having to explicitly exclude each assay that we cannot process. The pipeline has failed on incorrect datasets so far, but we don't want to rely on that and then accidentally process something in production that we didn't mean to!

From Ambrose:

> I believe that computationally, the pipelines are identically processed by Cellranger, which adds sequences to the reference that enable alignment of the the CITE-seq oligos that mark targeted proteins. 
> It seems logical to me that we would want to capture the protein targets being assayed by CITE-seq in the metadata. If this doesn't exist, I'm happy to chat cc'd metadata folks to make this happen. The presence of such fields would then positively label an experiment as a CITE-seq experiment.

**Comment by hewgreen** [comment]
Initially we were going to reuse the targets module to capture the proteins as Ambrose suggests. However, I think this module should be imaging specific and therefore should be nested into the imaging protocol schema. Nevertheless, the fields will be quite similar and the principle identical. We need two fields mapping readout sequence to protein identifier (uniprot). Again this will be an array.

**Comment by samanehsan** [comment]
Following up on the topic of queries:
> The goal here is to make sure the inclusion criteria of the query is specific enough to match only 10x v2 single-cell sequencing data without having to explicitly exclude each assay that we cannot process. 

For example, instead of excluding single nuclei sequencing assays, we are matching on "nucleic_acid_source: single cell" so that we only receive data from whole cells. @hewgreen @willrockout is there something equivalent we can do to exclude CITE-seq assays or is that metadata something that will be determined as a part of this ticket?

**Comment by willrockout** [comment]
@samanehsan that can be an outcome of this ticket. I believe the main goal is figuring out what we need to include metadata wise to make the data usable downstream such as antibody and barcode information.

**Comment by lauraclarke** [comment]
@willrockout you are correct sufficient information to ensure the data is analysable downstream is the main goal but if in the same stroke we can make the metadata more useful for the pipelines team to have accurate notifications then all the better

**Comment by hewgreen** [comment]
Thanks @samanehsan. Just thinking aloud, our pipelines are much less generic than we often refer to in conversation. Saying 'we have pipelines for 10x' is not right. For '10x', we only support whole cell, 10x v2 library prep, non CITE seq, non cell hashing, human etc. Do you have a tight specification anywhere in plain english about exactly what the pipelines can handle? Where they can be generic (such as sequencing platform) and where they require specific experimental methods. It may be good to systematically check these metadata fields as a whole. Although the caveats listed above touch on different parts of the metadata it would be good for us to spot opportunities to reuse and specify fields to avoid adding extra fields every time we discover tighter specificity is required.

**Comment by samanehsan** [comment]
Thanks @hewgreen. There is some information about what the pipelines support in the [HCA data portal](https://staging.data.humancellatlas.org/learn/userguides/data-processing-pipelines/overview-of-data-processing-pipelines-user-guides), and I also have some [materials](https://drive.google.com/drive/u/0/folders/1qOIyRKIr1mTuzoFZTUfX0p71ZAB-97ar) from the breakout session on this topic from the last DCP meeting. This is my current understanding of what the pipelines support, but any of the comp bios on the data pipelines team can offer more insight:

For 10x we support: 3’ scRNA-Seq, 10X v2 (and v3) library prep, non cell hashing from human donors

For SS2 we support: paired-end, plate-based, non-fluidigm, Smart-seq2 library prep from human donors

**Comment by willrockout** [comment]
@samanehsan I see you added v3 for the 10x pipeline does that mean you will be able to analyze CITE-seq data?

**Comment by samanehsan** [comment]
I'm not sure @willrockout, but @ambrosejcarr, @barkasn or @kishorikonwar could provide more information here

**Comment by malloryfreeberg** [comment]
For the CITEseq dataset already in the DS (and we are re-ingesting right now), the cell hashtag and antibody-derived tag library preps indicate they are prepping "polyA RNA". I don't think this is correct. Aren't these synthetic oligonucleotide sequences? 

If so, we need to find an ontology term that we can use to describe what is being sequence. I don't think an appropriate one exists in [ontology we use](https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0004446) yet. What about:
1. oligonucleotide (synonym: oligomer)
1. polynucleotide
1. synthetic oligonucleotide

Thoughts @daniwelter @willrockout @samanehsan @ambrosejcarr

**Comment by ambrosejcarr** [comment]
>  the cell hashtag and antibody-derived tag library preps indicate they are prepping "polyA RNA". I don't think this is correct. 

This is correct but potentially inadequately specific. The oligos are indeed poly-adenylated RNA; they are also synthetic oligonucleotides with poly-A tails. The oligos are constructed this way to make them amenable to capture by the same 10x v3 primers used to capture cellular mRNA, and the hybrid library is then co-created and co-sequenced.

**Comment by ESapenaVentura** [comment]
There are current discussions to capture CITE-Seq that overrule this conversation
